### DISCUSSION TITLE
pls help with AUDIOGRIDDER GUI parallels osx

### DISCUSSION THREAD

**Discussion by 3333qqq** [apohl79/audiogridder#558] [Help]
HI ! 
Im trying to use audiogridder beta 1.2 on my mabook pro 2015 mojave and logic 10.5 through parallels desktop - windows 10 from bootcamp partition - parallel toolbox activated and apparently working fine. 
i can open the plugin window on a track in logic, i can browse through some vst plugins and open them however i cant see the gui anywhere... nothing pops up in the windows 10/ parallel window and from the au plugin i can only see the parameters of the plugins as slides and numbers ... any idea on how to see the actual plugin graphic interface ? thanks a lot!!!!

**Comment by CaBuffalo** [comment]
Hi,

Go to the setting of AG plugin, disable the Generic editor which shows the
parameters of the plugin in/ This should fix the problem.

On Sun, May 16, 2021 at 7:58 AM 3333qqq ***@***.***> wrote:

> HI !
> Im trying to use audiogridder beta 1.2 on my mabook pro 2015 mojave and
> logic 10.5 through parallels desktop - windows 10 from bootcamp partition -
> parallel toolbox activated and apparently working fine.
> i can open the plugin window on a track in logic, i can browse through
> some vst plugins and open them however i cant see the gui anywhere...
> nothing pops up in the windows 10/ parallel window and from the au plugin i
> can only see the parameters of the plugins as slides and numbers ... any
> idea on how to see the actual plugin graphic interface ? thanks a lot!!!!
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/558>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6BC5JUUPUBNC7CNHDTTN7MRBANCNFSM4463I45A>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by 3333qqq** [comment]
yessss!!!! that finally got it working!!!! thanks so much!!!
just one more thing... in the au plugin window the gui is somehow zoomed in a lot and i can see just a portion of the interface.. how can i scale it down ? 
also... whats the best way to use this ? do you change the parameteres in the windows/parallels gui or through the plugin in logic ?

**Comment by CaBuffalo** [comment]
in the au plugin window the gui is somehow zoomed in a lot and i can see
just a portion of the interface.. how can i scale it down ?  --- it sounds
like the display property setting in windows. Make sure it's 100% scale.

whats the best way to use this ? do you change the parameteres in the
windows/parallels gui or through the plugin in logic ?  --- I always steam
the GUI to Logic and control the plugin from there.


On Sun, May 16, 2021 at 9:26 AM 3333qqq ***@***.***> wrote:

> yessss!!!! that finally got it working!!!! thanks so much!!!
> just one more thing... in the au plugin window the gui is somehow zoomed
> in a lot and i can see just a portion of the interface.. how can i scale it
> down ?
> also... whats the best way to use this ? do you change the parameteres in
> the windows/parallels gui or through the plugin in logic ?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/558#discussioncomment-744898>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6HVZRW6IU5B7BNE5A3TN7W4JANCNFSM4463I45A>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by 3333qqq** [reply]
please could you explain exaclty where i could change the 100% scale ? i had a look around but couldnt find any option 
thx

**Comment by CaBuffalo** [comment]
Go to the display setting of Windows 10. Check "scale and layout"

On Sun, May 16, 2021 at 10:33 AM 3333qqq ***@***.***> wrote:

> please could you explain exaclty where i could change the 100% scale ? i
> had a look around but couldnt find any option
> thx
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/558#discussioncomment-745064>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6C7FYBNNRUTDKZTZG3TN76W7ANCNFSM4463I45A>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by 3333qqq** [reply]
had a look but cant seem to find the scale and layout ... is it maybe the "resolution" i have it set to 2852 x 1678 9reccomended)

**Reply by 3333qqq** [reply]
ps. got it working!!! messed around a bit and found there was a 200% on txt (reccomended) changed that and it fixed it!!! thx a lot!!!!
